internal/cmd/testdata/scripts/root.txt

```diff
@@ -0,0 +1,17 @@
+# test that chezmoi apply uses .chezmoiroot
+chezmoi apply
+cmp $HOME/.file golden/.file
+
+# test that chezmoi add uses .chezmoiroot
+symlink $HOME/.symlink -> .file
+chezmoi add $HOME${/}.symlink
+cmp $CHEZMOISOURCEDIR/home/symlink_dot_symlink golden/symlink_dot_symlink
+
+-- golden/.file --
+# contents of .file
+-- golden/symlink_dot_symlink --
+.file
+-- home/user/.local/share/chezmoi/.chezmoiroot --
+home
+-- home/user/.local/share/chezmoi/home/dot_file --
+# contents of .file
```

Comment:
perhaps it worth it to add a test for `init` with a config template as well?

Reply:
Done (in my working copy, not pushed yet).